application/forms/EventRuleConfigElements/EscalationCondition.php

```diff
@@ -166,7 +161,9 @@ protected function assemble(): void
             $this->registerElement($col);
             $this->registerElement($op);
             $this->registerElement($val);
-            $removeButton = $this->createRemoveButton($i);
+            $removeButton = $conditionCount === 1 && ! $this->allowZeroConditions
```

Comment:
Please use `if` condition to keep it simple.
```suggestion
if (..) {
                $removeButton = $this->createRemoveButton($i);
                
                if ($removeButton->hasBeenPressed()) {
                    $removePosition = $i;
                }
            }
```